packages/perseus/src/widgets/dropdown/validate-dropdown.ts

```diff
@@ -0,0 +1,22 @@
+import type {PerseusScore} from "../../types";
+import type {PerseusDropdownUserInput} from "../../validation.types";
+
+/**
+ * Checks if the user has selected an item from the dropdown before scoring.
+ * This is shown with a userInput value / index other than 0.
+ * @param userInput
```

Comment:
I would suggest removing these two lines:

The first one documents the internal implementation of the function (how it checks validation) which you can see by reading the code. It also presents something that can easily get out of sync with the code. A great guide for comments that I was told one time was "document why, not what". :) 

The `@param userInput` doesn't really add any value.